.circleci/scripts/windows_cuda_install.sh

```diff
@@ -25,7 +25,7 @@ else
     exit 1
 fi
 
-if [[ "$cuda_major_version" == "11" && "${JOB_EXECUTOR}" == "windows-with-nvidia-gpu" ]]; then
+if [[ "$cuda_major_version" == "11" && "${JOB_EXECUTOR:-}" == "windows-with-nvidia-gpu" ]]; then
```

Comment:
This script will fail with an unbound error here if JOB_EXECUTOR doesn't exist.

JOB_EXECUTOR isn't defined on GHA so I'm just going to give it a default value of an empty string here.